test/common/upstream/upstream_impl_test.cc

```diff
@@ -2564,26 +2746,26 @@ TEST_F(ClusterInfoImplTest, ExtensionProtocolOptionsForFilterWithOptions) {
     TestNetworkFilterConfigFactory factory(factoryBase);
     Registry::InjectFactory<Server::Configuration::NamedNetworkFilterConfigFactory> registry(
         factory);
-    clusters.push_back(makeCluster(yaml));
+    clusters.push_back(makeCluster(yaml, false));
```

Comment:
I think these tests _should_ be converted to typed extension protocol options, there shouldn't be a need to boost though.

Reply:
@htuch made changes as per your suggestion, let me know if this is not ok.   anything else apart from this particular test case you want to convert in typed extension protocol options ?